static/src/javascripts-legacy/projects/common/utils/check-dispatcher.js

```diff
@@ -0,0 +1,39 @@
+define([
+    'common/utils/config',
+    'common/utils/check-mediator',
+    'common/modules/email/run-checks',
+    'common/modules/email/email-article',
+    'common/modules/experiments/ab-test-clash'
+], function(
+    config,
+    checkMediator,
+    emailRunChecks,
+    emailArticle,
+    clash
+) {
+
+    var checksToDispatch = {
+        isUserInAClashingAbTest: function() {
+            return clash.userIsInAClashingAbTest();
+        },
+        emailCanRun: function() {
+            return emailRunChecks.allEmailCanRun();
+        },
+        listCanRun: function() {
+            return find(emailArticle.getListConfigs(), emailRunChecks.listCanRun) ? true : false;
```

Comment:
i haven't read all of this PR yet but assuming `getListConfigs` is an array, you can use `Array.some`